src/Router/BaseRouter.php

```diff
@@ -106,37 +107,38 @@ public function handle(ServerRequestInterface $request) : ResponseInterface
             $path    = $uri->getPath();
             $baseurl = $uri->withPath("/")->withQuery("");
             switch (count($components)) {
-            case 1:
-                $request = $request
-                    ->withAttribute("baseurl", rtrim($baseurl->__toString(), '/'))
+                case 1:
+                    $request = $request
+                    ->withAttribute("baseurl", $baseurl->__toString())
```

Comment:
@johnsaigle Did you test before removing `rtrim` ?

Reply:
I don't remember since I made this PR 6 months ago 🤷‍♂ 

I'll add it back.